staging/src/k8s.io/code-generator/_examples/apiserver/openapi/zz_generated.openapi.go

```diff
@@ -30,66 +30,57 @@ import (
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
```

Comment:
these generated changes are also unexpected

Reply:
these are changed by hack/update-bazel.sh and hack/update-codegen.sh, or the CI will fail by hack/verify-bazel.sh and hack/verify-codegen.sh

Reply:
so I reverted these auto generated files, local check using hack/update-bazel.sh and hack/update-codegen.sh fails, but upstream looks fine. So yeah they are not required to pass the upstream CI. Thanks for pointing this out.